### PR TITLE
Lpa 3271 completed status bug, completed status is now explicitly checked

### PR DESCRIPTION
**Pull request by SeemaMenon**

### CODE REVIEW

File: module/Application/src/Controller/AbstractAuthenticatedController.php [hunk 1]
```diff
@@ -7,6 +7,8 @@
 use Application\Model\Service\Lpa\Application as LpaApplicationService;
 use Application\Model\Service\Session\SessionManager;
 use Application\Model\Service\User\Details as UserService;
+use Exception;
```
Comment (taylorsj1980) [inline module/Application/src/Controller/AbstractAuthenticatedController.php]: These 2 use statements aren't required

File: module/Application/src/Controller/AbstractAuthenticatedController.php [hunk 2]
```diff
@@ -74,6 +76,7 @@ public function __construct(
         $this->lpaApplicationService = $lpaApplicationService;
         $this->userService = $userService;
 
+
```
Comment (taylorsj1980) [inline module/Application/src/Controller/AbstractAuthenticatedController.php]: Please remove extra blank line

File: module/Application/src/Controller/AbstractAuthenticatedController.php [hunk 3]
```diff
@@ -170,6 +173,16 @@ public function onDispatch(MvcEvent $e)
         return $view;
     }
 
+    /**
+     * @return DateTime
+     *
+     */
+    public function getTrackFromDate()
```
Comment (taylorsj1980) [inline module/Application/src/Controller/AbstractAuthenticatedController.php]: Don't need this as a separate function. Please move the logic down into the area where it is called in the `statusDescriptionAction` in the `DashboardController`

File: module/Application/src/Controller/AbstractAuthenticatedController.php (hunk 1 above)
Comment (SeemaMenon) [reply module/Application/src/Controller/AbstractAuthenticatedController.php]: done

File: module/Application/src/Controller/AbstractAuthenticatedController.php (hunk 2 above)
Comment (SeemaMenon) [reply module/Application/src/Controller/AbstractAuthenticatedController.php]: done

File: module/Application/src/Controller/AbstractAuthenticatedController.php (hunk 3 above)
Comment (SeemaMenon) [reply module/Application/src/Controller/AbstractAuthenticatedController.php]: done

**Pull request merged**